src/youtube-player/youtube-player.ts

```diff
@@ -69,6 +69,9 @@ interface Player extends YT.Player {
 // The only field available is destroy and addEventListener.
 type UninitializedPlayer = Pick<Player, 'videoId' | 'destroy' | 'addEventListener'>;
 
+/** Whether we're currently rendering inside a browser. */
+const isBrowser = typeof window === 'object' && !!window;
```

Comment:
I guess usually we'd use `cdk/platform`, but here we don't want to add a dep on the `cdk`. Add a comment for this?

Reply:
Yep, I wanted to avoid the dependency. I'll add the comment.